db/system_keyspace.cc

```diff
@@ -564,7 +564,10 @@ schema_ptr system_keyspace::size_estimates() {
             {"partition_key", utf8_type}
         }, // CLUSTERING ORDER BY (partition_size DESC)
         // regular columns
-        {{"compaction_time", timestamp_type}},
+        {
+            {"row_count", long_type},
```

Comment:
`rows_count` reads better (as English)

Reply:
it absolutely doesn't, in proper English the first part of noun conjugation is never plural

Reply:
and field names are UI, as opposed to code

Reply:
(I'll make that just "rows" to avoid offending anybody's language intuitions, shall I...)

Reply:
> (I'll make that just "rows" to avoid offending anybody's language intuitions, shall I...)

ok

Reply:
@cmm, looks like you missed this change

Reply:
I'm being amazing today, probably need more sleep